contract/scripts/install-bundles.sh

```diff
@@ -3,8 +3,15 @@
 
 set -xueo pipefail
 
+SCRIPTS=$(dirname "${BASH_SOURCE[0]}")
+source "$SCRIPTS/wait-for-block.sh"
```

Comment:
`wait-for-block.sh` sounds like it will wait. it's just setting up some env. 

please either rename it something the implies it loads (e.g.`env-setup.sh` or `define-wait.sh` ) or make it do the waiting.